jenkins/ci.suse.de/cloud-mkcloud6.yaml

```diff
@@ -18,10 +18,8 @@
         - 'cloud-mkcloud{version}-job-backup-restore-{arch}'
         - 'cloud-mkcloud{version}-job-crowbar_register-{arch}'
         - 'cloud-mkcloud{version}-job-dvr-{arch}'
-        - 'cloud-mkcloud{version}-job-hyperv-{arch}'
         - 'cloud-mkcloud{version}-job-raid-{arch}'
         - 'cloud-mkcloud{version}-job-ssl-{arch}'
-        - 'cloud-mkcloud{version}-job-upgrade-{arch}'
```

Comment:
Are we sure about this?

Reply:
Isn't this about upgrading from 5 to 6? Do we still need that?

Reply:
Ah, if so, than it's fine